dmutils/dmp_so_status.py

```diff
@@ -1,8 +1,8 @@
 from datetime import datetime
 
-DOS6_OPEN = datetime(2022, 1, 14, 12, 0, 0)
-DOS6_CLOSE = datetime(2022, 2, 24, 15, 0, 0)
+G13_OPEN = datetime(2022, 3, 9, 10, 0, 0)
+G13_CLOSE = datetime(2022, 5, 18, 15, 0, 0)
```

Comment:
Is this BST or UTC?

Reply:
I'm not sure, but there's no clock going forwards and backwards between DOS6 and G13, which means it won't matter which (whoever did this before me, will be working on the same BST or UTC times)

Reply:
if you agree with that?

Reply:
(if i've understood you correctly, that is)

Reply:
I believe this is on our GMT/BST times, as G13 has the same times (not dates) as DOS6, and the entry was `15, 0, 0` for DOS6, and we know that this was 3pm our time on the 24th Feb

Reply:
also, i think at the moment GMT = UTC, we're not on BST right now

Reply:
At the moment, you're correct that BST == UTC. However, will this still be true in May when G13 closes?

Reply:
oooooohhh snap, I was so focused on the open times (and that it fell before clock changes)! Excellent spot!

Reply:
Fixed!

Reply:
I've just looked at the API docs (https://docs.python.org/3/library/datetime.html#datetime.datetime), and I think what you've got here works fine.

> [](https://docs.python.org/3/library/datetime.html#datetime.datetime.now)classmethod datetime.now(tz=None)
> Return the current local date and time.

It looks like both `datetime.datetime` and `datetime.now` default to the local timezone.

Reply:
@tberey - see my previous comment. I think you need to either:

1. set the closing time to 3pm and leave tzinfo=None (the default)
2. set the closing time to 4pm and set tzinfo=UTC (?)

Reply:
Oh, in python trying to be convenient, that's really quite inconvenient that is does that - everything I've worked with uses UTC, although I guess we're lucky 50% of the time that lines up with GMT/BST

Reply:
@bjgill I've left as the default, i.e. tzinfo=None